version.sbt

```diff
@@ -1 +1 @@
-version in ThisBuild := "0.6.6-SNAPSHOT"
+version in ThisBuild := "0.6.6"
```

Comment:
you shouldn't manually change this.

Just do `> release` in sbt

Reply:
That change was produced by `> release` command